src/Advection/upwind_biased_first_order.jl

```diff
@@ -0,0 +1,33 @@
+#####
+##### Upwind-biased 1rd-order advection scheme
+#####
+
+struct UpwindBiasedFirstOrder <: AbstractUpwindBiasedAdvectionScheme{1} end
```

Comment:
The boundary buffer here ("1") is inconsistent with the boundary buffer given below ("0")

Reply:
Thinking about it, I think the boundary buffer should be 1 because we need to limit to a centered second-order difference on boundaries.

Reply:
I changed the `1` to a `0`

Reply:
That makes sense.  Just to confirm, do you mean we should have the following?

```
struct UpwindBiasedFirstOrder <: AbstractUpwindBiasedAdvectionScheme{1} end

@inline boundary_buffer(::U1) = 1
```

Reply:
Yes.

Reply:
Great!  Now pushed.